compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs

```diff
@@ -865,7 +865,12 @@ checkDefTypeSyn DefTypeSyn{synParams,synType} = do
 -- | Check that an interface definition is well defined.
 checkIface :: MonadGamma m => Module -> DefInterface -> m ()
 checkIface m iface = do
-  let tcon = Qualified PRSelf (moduleName m) (intName iface)
+  
+  -- check view
+  tycon <- match _TCon (EExpectedViewType viewtype) viewtype
+  DefDataType _loc _naem _serializable tparams dataCons <- inWorld (lookupDataType tycon)
```

Comment:
```suggestion
  DefDataType _loc _naem serializable tparams dataCons <- inWorld (lookupDataType tycon)
```

I think we also want to check that `serializable == IsSerializable True`

Reply:
that should happen in `compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs`, though

Reply:
sorry, this is already checked in the module I linked x.x

Reply:
It is already in `compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs`